src/app/voice-assistant/voice-assistant-chat/voice-assistant-chat.component.ts

```diff
@@ -90,7 +97,14 @@ export class VoiceAssistantChatComponent implements OnInit {
                 Validators.minLength(2),
                 Validators.maxLength(255),
             ]);
+            this.toggleDeleteChat(this.chatService.chats);
```

Comment:
This line can be removed or does this line has an oppose, which I can not see right now?